test/colab/colab-test.mk

```diff
@@ -0,0 +1,38 @@
+ifdef CI
+	DOCKER_RUN = @docker run -v $(shell pwd):/usr/src/app:ro -v $(shell pwd)/dist:/usr/src/app/dist:rw --rm --platform=linux/amd64 -e LAYER_API_KEY=$(shell cat .test-token) --name colab-test $(DOCKER_IMAGE_NAME)
+else
+	DOCKER_RUN = @docker run -v $(shell pwd):/usr/src/app:ro -v $(shell pwd)/dist:/usr/src/app/dist:rw --rm --platform=linux/amd64 -e LAYER_API_KEY=$(shell cat .test-token) --name colab-test -it $(DOCKER_IMAGE_NAME)
+endif
+
+
+.PHONY: colab-test-internal
+colab-test-internal: $(TEST_TOKEN_FILE) $(COLAB_TEST_HOME)/test_import_login_init.ipynb colab-test-pull
+	$(DOCKER_RUN)
+
+.PHONY: colab-test-internal-local
+colab-test-internal-local: $(TEST_TOKEN_FILE) $(COLAB_TEST_HOME)/test_import_login_init.ipynb colab-test-build
+	$(DOCKER_RUN)
+
+.PHONY: colab-test-pull
+colab-test-pull:
+	@docker pull $(DOCKER_IMAGE_NAME)
+
+.PHONY: colab-test-build
+colab-test-build: $(COLAB_IMAGE_BUILD_STAMP)
+$(COLAB_IMAGE_BUILD_STAMP): $(COLAB_TEST_HOME)/requirements-fixed.txt test/colab/Dockerfile
+	@docker build -t $(DOCKER_IMAGE_NAME) -f test/colab/Dockerfile .
```

Comment:
On a random ubuntu I was testing which did not had buildkit enabled it errors out complaining about it. We could `DOCKER_BUILDKIT=1` or maybe just add as a requirements to use it locally.

Reply:
Fair, I assumed that was now the default, but maybe it's not, thanks!